src/utils/matrix.ts

```diff
@@ -184,15 +130,13 @@ export async function initClient(
     fallbackICEServerAllowed: fallbackICEServerAllowed,
   });
 
-  try {
-    await client.store.startup();
-  } catch (error) {
-    logger.error(
-      "Error starting matrix client store. Falling back to memory store.",
-      error,
-    );
-    client.store = new MemoryStore({ localStorage });
-    await client.store.startup();
```

Comment:
Ah. You've fallen into the trap of confusing `client.store` with `client.cryptoStore`. This is an understandable mistake, because the name is unclear and the documentation is, well, about as good as the rest of the documentation for the js-sdk.

`client.store` is used to stash things like sync results. It's totally independent of the cryptostore, and uses a separate indexeddb database.

Anyway, calling `client.store.startup()` is important, and you need to keep this, otherwise `client.startClient()` will hang. (You might reasonably ask why `client.startClient` doesn't call `store.startup()` itself. I don't know.)

Reply:
Thank you so much Rich!!

This would have been a debugging session to fix this without your comment.

🙏

Reply:
Added: https://github.com/element-hq/element-call/pull/2591